driver/conv_driver.hpp

```diff
@@ -426,20 +439,48 @@ bool ConvDriver<Tgpu, Tref>::IsInputTensorTransform() const
 template <typename Tgpu, typename Tref>
 int ConvDriver<Tgpu, Tref>::ParseCmdLineArgs(int argc, char* argv[])
 {
+
     inflags.Parse(argc, argv);
 
     // try to set a default layout value for 3d conv if not specified from cmd line
     int spatial_dim = inflags.GetValueInt("spatial_dim");
 
     const std::string default_layout = (spatial_dim == 2) ? "NCHW" : "NCDHW";
 
+    //inflags value is empty, default value is used
+    //if it is supplied via cmd line, check the value.
     if(inflags.GetValueStr("in_layout").empty())
+    {
         inflags.SetValue("in_layout", default_layout);
+    }
+    else
+    {
+       std::string in_layoutValue = inflags.GetValueStr("in_layout");
+       ValidateLayoutInputParameters(in_layoutValue);
```

Comment:
https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1316#discussion_r767329617 seems unresolved

Reply:
@atamazov - "in_layout" name should be renamed?